website/content/docs/connect/proxies/envoy-extensions/configuration/otel-access-logging.mdx

```diff
@@ -0,0 +1,390 @@
+              Namespace = "<namespace containing the upstream service>"
+              Partition = "<partition containing the upstream service>"
+            }
+            URI = "<URI of the upstream service>"   
+            Timeout = "1s"
+        }
+        Authority = "<authority header to send in the gRPC request>"     
+        InitialMetadata = [
+            "<Key>" : "<value>"
+        ]
+      }
+      BufferFlushInterval = "1s"
+      BufferSizeBytes = 16384
+      FilterStateObjectsToLog = [
+        "Additional filter state objects to log in filter_state_objects"
+      ]
+      RetryPolicy = {
+        RetryBackOff = {
+            BaseInterval = "1s"
+            MaxInterval = "30s"
+        }
+        NumRetries = <uint32 value specifying the max number of retries>
+      }
+      Body = "Log Request Body"
+      Attributes = {
+         "<Key>" : "<value>"
+      }
+      ResourceAttributes = {
+         "<Key>" : "<value>"
+      }
+```
+
+## Specification
+
+This section provides details about the fields you can configure for the OpenTelemetry Access Logging extension.
+### `Name`
+
+Specifies the name of the extension. Must be set to `builtin/otel-access-logging`.
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: String value set to `builtin/otel-access-logging`.
+
+### `Arguments`
+
+Contains the global configuration for the extension.
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: Map
+
+### `Arguments.ProxyType`
+
+Specifies the type of Envoy proxy that this extension applies to. The extension only applies to proxies that match this type and is ignored for all other proxy types. The only supported value is `connect-proxy`.
+
+#### Values
+
+- Default: `connect-proxy`
+- This field is required.
+- Data type: String
+
+### `Arguments.ListenerType`
+
+Specifies the type of listener the extension applies to. The listener type is either `inbound` or `outbound`. If the listener type is set to `inbound`, Consul applies the extension so the access logging is enabled when other services in the mesh send messages to the service attached to the proxy. If the listener type is set to `outbound`, Consul applies the extension so the access logging is enabled when the attached proxy sends messages to other services in the mesh.
+
+#### Values
+
+- Default: inbound
+- This field is required.
+- Data type is one of the following string values:
+  - `inbound`
+  - `outbound`
+
+### `Arguments.Config`
+
+Contains the configuration settings for the extension.
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: Map
+
+### `Arguments.Config.LogName`
+
+Specifies the friendly name of the access log to be returned in StreamAccessLogsMessage.Identifier. This allows the access log server to differentiate between different access logs coming from the same Envoy.
+
+#### Values
+
+- Default: value given to ListenerType
```

Comment:
```suggestion
- Default: Inherits the value given to ListenerType
```
We're still firming this up so I'm leaving this for now, but I think we're going to standardize on "None" as the default even when the parameter inherits a value from another configuration. This is to differentiate from true default values programmed into the software that you must explicitly override. We'll describe inheritance in the description of the parameter. Does that make sense?

Reply:
Used the suggested way. Please check if the updated one sounds good.